src/yocto/recipes.md

```diff
@@ -0,0 +1,186 @@
+# Recipes
+Official page: https://docs.yoctoproject.org/overview-manual/concepts.html#recipies
+
+## Recipes files
+Recipe files or also known as BB files, are files that contain information and instructions that build tool (bitbake) takes for generating the packages. This information for instructions can be resumed as:
+1. Descriptive information about package
+2. Section information
+2. The recipe version
+3. The Licience
+3. Existing dependencies 
+4. Where the source code resides and how to fetch it 
+5. Whether the source code requires any patches, where to find them, and how to apply them.
+6. How to configure and compile the source code.
+7. Where on the target machine to install the package or packages created.
+## Variables for bb files
+https://docs.yoctoproject.org/dev/ref-manual/variables.html
```

Comment:
If you put a link then maybe instead only setup an example, we try to have as little redundant as possible on topics that are well explain in yocto. Then just complement information.